stock_quant_yahoo_product_state/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2017-2018 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Quant Yahoo Product State",
+    "version": "11.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "category": "Stock",
+    "license": "AGPL-3",
+    "depends": [
+        "stock",
+        "stock_quant_list_view",
```

Comment:
@kanda999 Please check if we should reverse the dependencies. In fact, please check if we can get rid of stock_quant_list_view by delegating necessary changes on stock.quant list to other existing modules.

Reply:
@AungKoKoLin1997 
We can remove this module by moving stock_quant.py to product_yahoo_auction_sst and adjusting the view in stock_quant_list_view.
stock_quant_list_view contains product_ext_sst as a dependency, so product_yahoo_aution_sst is installed.